File: CPP/Program/Baju.cpp
#include <iostream>
#include <string>

#include "Aksesoris.cpp"

class Baju : public Aksesoris{
    private:
        string untukHewan;
        string size;
        string merk;
    public:
        Baju(string id, string nama, int harga, int stok) : Aksesoris(id, nama, harga, stok){
            this->untukHewan = "";
            this->size = "";
            this->merk = "";
        }

        void setUntukHewan(string untukHewan){
            this->untukHewan = untukHewan;
        }
        string getUntukHewan(){
            return this->untukHewan;
        }

        void setSize(string size){
            this->size = size;
        }
        string getSize(){
            return this->size;
        }

        void setMerk(string merk){
            this->merk = merk;
        }
        string getMerk(){
            return this->merk;
        }


        ~Baju(){}
};

File: PHP/Program/Baju.php
<?php
require_once "Aksesoris.php";

class Baju extends Aksesoris{
    private $untukHewan;
    private $size;
    private $merk;
    private $foto;

    public function __construct($id, $nama, $harga, $stok, $jenis, $bahan, $warna, $untuk, $size, $merk, $foto) {
        parent::__construct($id, $nama, $harga, $stok, $jenis, $bahan, $warna);
        $this->untukHewan = $untuk;
        $this->size = $size;
        $this->merk = $merk;
        $this->foto = "default.jpg";
    }

    public function setUntukHewan($untuk){
        $this->untukHewan = $untuk;
    }
    public function getUntukHewan(){
        return $this->untukHewan;
    }

    public function setSize($size){
        $this->size = $size;
    }
    public function getSize(){
        return $this->size;
    }

    public function setMerk($merk){
        $this->merk = $merk;
    }
    public function getMerk(){
        return $this->merk;
    }

    public function getFoto(){
        return $this->foto;
    }
}